tests/summarycode/data/package_summary/basic-plugin-testing/expected.json

```diff
@@ -0,0 +1,39 @@
+{
+  "files": [
+    {
+      "path": "codebase",
+      "type": "directory",
```

Comment:
What is the use of this test? 
There are no package manifests, so no packages will be detected. What would you calculate a summary for?

Reply:
Removed this as this is of no use now